src/main/java/net/minecraftforge/registries/tags/IReverseTag.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.registries.tags;
+
+import net.minecraft.core.Holder;
+import net.minecraft.tags.TagKey;
+
+import java.util.stream.Stream;
+
+/**
+ * A reverse tag is an object aware of what tags it is contained in.
+ * {@link Holder}s implement this interface.
+ */
+public interface IReverseTag<V>
```

Comment:
Persists relative to what?

Reply:
Where it is gotten from or through reloads? I don't know, that is why I presume the answer is no

Reply:
I think it's easiest to specify that reverse tags make no guarantee to their persistence. Mods should be looking up a reverse tag every time they need one rather than storing them.